python/pyspark/sql/dataframe.py

```diff
@@ -861,11 +861,11 @@ def groupBy(self, *cols):
             Each element should be a column name (string) or an expression (:class:`Column`).
 
         >>> df.groupBy().avg().collect()
-        [Row(avg(age)=3.5)]
+        [Row(avg_c1=3.5)]
```

Comment:
What's the problem with it? Expecting to see special chars in aggregation attribute but it doesn't?